app/example-page/page.tsx

```diff
@@ -0,0 +1,11 @@
+import ExampleComponent from '../../components/ExampleComponent/ExampleComponent';
+
+const ExamplePage = () => {
```

Comment:
Arrow functions look good to me, `functions` syntax is okay too, i don't mind which one i use